include/config.h

```diff
@@ -18,6 +18,13 @@ enum mako_button_binding {
 	MAKO_BUTTON_BINDING_INVOKE_DEFAULT_ACTION,
 };
 
+enum mako_sort_criteria {
+	MAKO_SORT_CRITERIA_TIME = 1,
+	MAKO_SORT_CRITERIA_URGENCY = 2,
+	MAKO_SORT_ASC_TIME = 1,
```

Comment:
It's pretty weird to have multiple enum items with the same value. I think the last two can be removed.